database/dataGen.js

```diff
@@ -0,0 +1,34 @@
+const faker = require('faker');
+
+const genRecommendation = (amount) => {
+  const recommendations = [];
+  for (let i = 0; i < amount; i += 1) {
```

Comment:
Would rename ```amount``` to ```count```. Also you might want to consider randomizing the number of recommendations and listings, for testing purposes